### ISSUE TITLE
Vertical Portals like keeping you in an infinite loop

### ISSUE THREAD

**Issue by FungletheBungle** [MattJeanes/world-portals#24]
As the title states, when I attempted in hammer to create a vertical portal, it decided I should stick around a bit longer then I would have liked.

Find video at this link https://youtu.be/8a4Blt-s5g0

**Comment by FungletheBungle** [comment]
In addition, in much rarer cases, it forgets about teleporting you and shoots you under the portal, leaving you stuck in whatever is underneath it.

**Comment by FungletheBungle** [comment]
Testing with Grenade:https://youtu.be/9-BREiF9Aio

**Comment by MattJeanes** [comment]
Yeah this is a known issue, portals do not like being vertical for some reason. It may be some time before I can look at this so if you feel like this is something you can fix in the code I'd happily accept a PR 🙂

**Comment by AmyJeanes** [comment]
Fixed.
